configs/setup/kube.json

```diff
@@ -2,7 +2,7 @@
 	"K8sVersion": "1.25.9",
 	"AlternativeImageRepo": "",
 	"ApiserverAdvertiseAddress": "",
-	"PodNetworkCidr": "192.168.0.0/16",
+	"PodNetworkCidr": "10.168.0.0/16",
```

Comment:
Can you also try to change these addresses on loader side? So we won't need to change them here.